src/store/plugins/usersWebSocketPlugin.js

```diff
@@ -3,41 +3,54 @@ const RECONNECT_TIME = 2000;
 
 export default function createUsersWebSocket() {
   return function connect(store, unsubscribeCallback) {
+    // unsubscribe a store subscription made in the previous call
```

Comment:
either we can remove this or we need to change this - I don't this anyone reading this would understand what this line means.